File: ggp/Sources/ggp/models/Classifier.swift
import Foundation

struct Classifier: Decodable {
    let name: String
    let importPackage: String
    let parameters: [Parameter]

    enum CodingKeys : String, CodingKey {
        case name
        case importPackage = "import_package"
        case parameters
    }

    struct Parameter: Decodable{
        let name: String
        let value: String
    }
}

struct ClassifierList: Decodable{
    let classifiers : [Classifier]
}

File: ggp/src/main/kotlin/models/Classifier.kt
package models

import com.google.gson.annotations.SerializedName

data class Classifier(
    override val name: String,
    @SerializedName("import_package")
    override val importPackage: String,
    override val parameters: List<String> = emptyList()
) : Component()

data class ClassifierList(
    val classifiers: List<Classifier> = emptyList()
)